spring-cloud-dataflow-server-local/src/main/resources/dataflow-server.yml

```diff
@@ -15,7 +15,7 @@ security:
 
 spring:
   application:
-    name: spring-cloud-dataflow-server
+    name: spring-cloud-dataflow-server-local
```

Comment:
That was my line of thinking (consistency meaning they all use the same name "spring-cloud-dataflow-server" which does describe what this is, in terms of its role). But perhaps the tradeoff favors indicating the implementation within the name. I'd like to hear @ericbottard's thoughts.